backends/arm/operators/op_select.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2024 Arm Limited and/or its affiliates.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import List
+
+import serializer.tosa_serializer as ts
+from executorch.backends.arm.operators.node_visitor import (
+    NodeVisitor,
+    register_node_visitor,
+)
+
+from executorch.backends.arm.tosa_mapping import TosaArg
+
+from executorch.backends.arm.tosa_utils import build_reshape, tosa_shape
+from serializer.tosa_serializer import TosaOp
+from torch.fx import Node
+
+
+@register_node_visitor
+class SelectVisitor(NodeVisitor):
+    target = "aten.select_copy.int"
```

Comment:
I guess we can't replace this with slice easily in a pass?

Reply:
Hi, 

Thanks for review!
We could use a pass for this, but there is some work that is on-going that should enable this going forward.
I hope this helps!

Thanks, 
Saoirse